ops/docker/ci-builder/Dockerfile

```diff
@@ -157,3 +157,15 @@ COPY --from=rust-build /root/.rustup /root/.rustup
 # copy the rust installation, alongside the installed toolchains
 COPY --from=rust-build /root/.cargo/bin /root/.cargo/bin
 COPY --from=rust-build /root/.rustup /root/.rustup
+
```

Comment:
Don't change this Dockerfile

Reply:
1. Create new folder with new dockerfile
2. Create a Dockerfile that's mainly empty but start from the ci-builder at a given version.
3. Then add nvm in the new Dockerfile

Reply:
Make sure that I pass the platform when I am building locally. 

`--platform linux/amd64`

Reply:
```
docker buildx bake \                                                                              
    --progress plain \
    --platform linux/amd64 \
    --load \
    -f docker-bake.hcl \
    ci-builder-security-tools
```

Reply:
docker buildx bake \
    --progress plain \
    --load \
    -f docker-bake.hcl \
    --platform linux/amd64 \
    ci-builder-security-tools

Reply:
`docker buildx bake --progress plain --load -f docker-bake.hcl --platform linux/amd64 ci-builder-security-tools`